app/src/lib/Map.svelte

```diff
@@ -50,8 +50,27 @@
         return div
     }
 
+    const getUserLocation = () => {
+        setInterval(() => {
```

Comment:
This returns a reference to the interval, you'll want to clean up the infinite loop in the `destroy` callback (line 85).

Reply:
how would I go about this?

Reply:
simplest way would be to have a userLocation component level variable that you assign inside `initilizeUserTracking` then call `clearInterval` on it in destroy